web/package.json

```diff
@@ -173,7 +173,7 @@
     "storybook": "^7.0.0-alpha.47",
     "tailwindcss": "3.1.8",
     "ts-node": "^10.9.1",
-    "typescript": "4.8.4",
+    "typescript": "5.0.4",
```

Comment:
Why is typescript updated? This is also unrelated to ESlint and should be made in a seperate PR if needed.

Reply:
fixed